internal/util/debug/debug.go

```diff
@@ -20,38 +20,60 @@ import (
 	"context"
 	_ "expvar" // for metrics
 	"fmt"
+	"log"
+	"math/rand"
+
 	"net"
 	"net/http"
 	_ "net/http/pprof" // for profiling
 	"slices"
 	"text/template"
 	"time"
 
+	"github.com/FerretDB/FerretDB/internal/util/must"
 	"github.com/arl/statsviz"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	io_prometheus_client "github.com/prometheus/client_model/go"
 	"go.uber.org/zap"
 	"golang.org/x/exp/maps"
-
-	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
+type metric struct {
+	metricName       string
+	metricType       int
+	metricLabel      string
+	metricLabelValue string
+}
+
 // RunHandler runs debug handler.
 func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *zap.Logger) {
 	stdL := must.NotFail(zap.NewStdLogAt(l, zap.WarnLevel))
 
-	http.Handle("/debug/metrics", promhttp.InstrumentMetricHandler(
+	metricHandler := promhttp.InstrumentMetricHandler(
 		r, promhttp.HandlerFor(prometheus.DefaultGatherer, promhttp.HandlerOpts{
 			ErrorLog:          stdL,
 			ErrorHandling:     promhttp.ContinueOnError,
 			Registry:          r,
 			EnableOpenMetrics: true,
 		}),
-	))
+	)
+	prometheus_metrics, err := prometheus.DefaultGatherer.Gather()
+	fmt.Printf("prometheus_metrics: %v\n", prometheus_metrics)
+	if err != nil {
+		log.Fatalf("error in Gathering prometheus metrics: %v", err)
+	}
```

Comment:
We should not use `fmt.Printf` or `log.Fatalf` there

Reply:
would `panic(err)` be fine instead of `log.Fatalf`

Reply:
I think we should be consistent with surrounding code that uses `must.NoError`